pinot-core/src/main/java/org/apache/pinot/core/common/MinionConstants.java

```diff
@@ -67,6 +67,7 @@ public static class ConvertToRawIndexTask {
   // Purges rows inside segment that match chosen criteria
   public static class PurgeTask {
     public static final String TASK_TYPE = "PurgeTask";
+    public static final String DELTA_TIME_PERIOD_KEY = "deltaPurgeTimePeriod";
```

Comment:
@jackjlli Can you please check the key used in LinkedIn? We want to make them the same so that you can eventually move to the open source implementation without causing backward incompatible issue

Reply:
Integration test added. Was hard to manipulate segmentMetadata in the setUp. I also was forced to implement a purge (does not purge return false every row and a BIG TODO in the baseMinionStarter ) to run the tests. 

Open to review again @Jackie-Jiang  many thx for you time